org.kde.kontact.json

```diff
@@ -0,0 +1,1224 @@
+                        "/bin/sqlite3"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "kmime",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/applications/18.08.1/src/kmime-18.08.1.tar.xz",
+                    "sha256": "dd14d7813199f4f859cb112dc34113269196c62fd9f05359e889913eb9d55d02"
+                }
+            ]
+        },
+        {
+            "name": "akonadi-mime",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/applications/18.08.1/src/akonadi-mime-18.08.1.tar.xz",
+                    "sha256": "af606127e9454568e9e8554942c5102d5eefe6b6a3875e1579c583e1d62cae13"
+                }
+            ]
+        },
+        {
+            "name": "kcontacts",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/applications/18.08.1/src/kcontacts-18.08.1.tar.xz",
+                    "sha256": "a2648dd9fc0d3232268ac291c47991f81a637226bfe3bc08c21ed6640fcf0df8"
+                }
+            ]
+        },
+        {
+            "name": "grantlee",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/steveire/grantlee/archive/v5.1.0.tar.gz",
+                    "sha256": "3836572fe5e49d28a1d99186c6d96f88ff839644b4bc77b73b6d8208f6ccc9d1"
+                }
+            ]
+        },
+        {
+            "name": "prison",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/frameworks/5.50/prison-5.50.0.tar.xz",
+                    "sha256": "4d98254c787f4ec4f80317a738d4f5ec6f2125144df1185fe61c444a086ec2ec"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "qrencode",
```

Comment:
This needs to be fixed for aarch64 exactly like this: https://github.com/flathub/cx.ring.Ring/commit/b024093f5115e874a567a5c7d049fbc70efea839#diff-2e592000f12516bca751f85f029e3ee3R169

Reply:
Or you could try cmake.

Reply:
that would actually likely be better!

Reply:
Oki, switched to cmake in qrencode. Let's hope that will do it :)